### DISCUSSION TITLE
添加OCR功能

### DISCUSSION THREAD

**Discussion by ghost** [immersive-translate/immersive-translate#548] [Ideas]
### 新功能 | New Feature

1.OCR图片上的文字，并且可以在一个文本框编辑
~~2.很多人没有ChatGPT的付费API，https://github.com/josStorer/chatGPTBox 这个就可以模拟网页访问GPT~~

### 为什么您想要添加这些功能 | Why you want to add these features

1.有些文字在图片上，有ocr方便很多。https://github.com/hcfyapp/crx-selection-translate 就有OCR，但是不开源，而且付费。
~~2.真没API~~

### 提议 | Proposal

https://tesseract-ocr.github.io/ 是开源OCR库。
~~网页API可以参考https://github.com/josStorer/chatGPTBox~~

**Comment by 1PercentSync** [comment]
chatgpt api请在开发者设置里开启beta

**Reply by ghost** [reply]
好的
